rdb/src/main/java/org/openmbee/sdvc/rdb/config/DatabaseDefinitionService.java

```diff
@@ -79,6 +77,16 @@ public boolean createProjectDatabase(Project project) throws SQLException {
         return !created.isEmpty();
     }
 
+    public void deleteProjectDatabase(Project project) throws SQLException {
+        Connection connection = crudDataSources.getDataSource("DEFAULT").getConnection();
```

Comment:
DEFAULT should use the ContextObject.DEFAULT_PROJECT from core
should wrap this in a try with resources/close connection per LGTM warning

Reply:
updated